packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2870,18 +2923,10 @@ export interface ErrorBarStrings {
 // @public
 export type ErrorType = keyof ErrorBarStrings;
 
-// @beta
-export interface FileDownloadError {
-    errorMessage: string;
-}
-
-// @beta
-export type FileDownloadHandler = (userId: string, fileMetadata: AttachmentMetadata) => Promise<URL | FileDownloadError>;
-
 // @beta
 export interface FileSharingOptions {
```

Comment:
Is this intended to still be called FileSharingOptions?

Reply:
yes otherwise upload will be broken

Reply:
And by that, you mean that the current beta is using this and we want to rename this _after_ the next beta release?

Reply:
no i mean to introduce attachmentOptions here, we need to work on upload refactor, which i am planning for another PR